src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  font-size: 30px;
+  padding: $square-paddings;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $square-spacing;
```

Comment:
The variable `$square-spacing` is used here. Ensure this variable is defined in your SCSS files to avoid compilation errors.